rubicon_ml/sklearn/pipeline.py

```diff
@@ -112,9 +117,13 @@ def score(self, X, y=None, sample_weight=None):
         sample_weight : list, optional
             If not None, this argument is passed as sample_weight keyword argument to the
             score method of the final estimator.
+        experiment: None
+            the experiment to log the score to. If no experiment is provided the score is logged to a new experiment with self.experiment_kwargs
         """
         score = super().score(X, y, sample_weight)
-
+        if experiment is None:
+            experiment = self.project.log_experiment(**self.experiment_kwargs)
+        self.experiment = experiment
```

Comment:
to fix dashboard experiments we need an extra default case:
1 - if experiment is not none, use the passed in experiment
2 - if experiment is none, but self.experiment is not none, use self.experiment
3 - if experiment is none, and self.experiment is none, log a new one